src/docs/_includes/head.html

```diff
@@ -0,0 +1,19 @@
+<head>
```

Comment:
Since now we have two layouts - one for component pages and one for index page - I moved the header to a separate template (because it's used by both layouts).